apps/www/src/lib/registry/default/ui/chart-area/AreaChart.vue

```diff
@@ -41,6 +41,8 @@ const emits = defineEmits<{
 type KeyOfT = Extract<keyof T, string>
 type Data = typeof props.data[number]
 
+const chartRef = Math.random().toString(36).substring(2, 9)
```

Comment:
`useId` might be a better way

Reply:
tried that first but could not get the gradient to find the right ID for the color resulting in all gray gradiants. Hence the solution with a unique ref for each chart.